app/internal/dynamo/client.go

```diff
@@ -23,17 +24,39 @@ func NewClient(cfg aws.Config, tableName string) (*Client, error) {
 	return &Client{table: tableName, svc: dynamodb.NewFromConfig(cfg)}, nil
 }
 
-func (c *Client) Get(ctx context.Context, id string, v interface{}) error {
-	keyID, err := attributevalue.Marshal(id)
+func (c *Client) GetAll(ctx context.Context, pk string, v interface{}) error {
+	pkey, err := attributevalue.Marshal(pk)
+	if err != nil {
+		return err
+	}
+
+	response, err := c.svc.Query(ctx, &dynamodb.QueryInput{
+		TableName:                 aws.String(c.table),
+		ExpressionAttributeNames:  map[string]string{"#0": "PK"},
+		ExpressionAttributeValues: map[string]types.AttributeValue{":0": pkey},
+		KeyConditionExpression:    aws.String("#0 = :0"),
```

Comment:
there is an expression builder to construct these, but :shrug: it seemed about as clear what was happening, and at least this matches the docs

Reply:
If I understand this right, we've set the value of `#0` to `"PK"` (Primary Key?) in `ExpressionAttributeNames` and the value of `:0` to `pkey` and we're then just setting a straight equality check in `KeyConditionExpression`? Could `#0` and `:0` be any string? It seemed that way from the docs but wanted to check as I've not worked with dynamoDB before.

Reply:
I guess I should not have followed whatever thing I saw verbatim and called them `#PK` and `:PK` instead

Reply:
😁 may be slightly clearer